configyaml/config/__init__.py

```diff
@@ -1,5 +1,5 @@
-from .base import ConfigBase
-from .base_list import ConfigBaseList
-from .base_dict import ConfigBaseDict
-from .base_wildcard_dict import ConfigBaseWildcardDict
-from .base_choice import ConfigBaseChoice
+from .base import AbstractNode
+from .list import NodeList
+from .dict import NodeDict
+from .wildcard_dict import WildcardDict
+from .choice import NodeChoice
```

Comment:
That seems more readable to me.

That's what I figured you were going for with `WildcardDict` -- should we separate it (and other more complete ones like it) out into a different module? I guess we could just lump them all in there for now, and move stuff around if we add other classes like `WildcardDict`? The only reason it threw me is because it's alongside all the others, but named differently...

Should these be in `nodes` or something?